src/Moryx.AbstractionLayer.Products.Endpoints/ProductManagementController.cs

```diff
@@ -99,6 +102,7 @@ private object ConvertParametersBack(string importerName, Entry currentParameter
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [Route("types")]
+        //todo why does this even exist. Find permission
```

Comment:
I don´t get the todo. Why you don´t just define a premission for it. I noticed that this permission is missing in the permission concept for the products endpoint

Reply:
I added this to the Permission CanEditType